src/cmd/health.js

```diff
@@ -60,6 +63,10 @@ const health_errors = {
         error_code: 'RSYSLOG_SERVICE_FAILED',
         error_message: 'RSYSLOG service is not started properly, Please verify the service with status command.',
     },
+    SYSLOG_NG_SERVICE_FAILED: {
```

Comment:
I couldnt find a place where this constant is using.

Reply:
That's right, I'll use it in the refactoring PR, currently inside the service check section we can find an object with 4 properties -  { name: rsyslog, pid, status: 'active'/'incative', error_type: persistent/temporary }
since we remove the rsyslog/syslog_ng error from the overall health status I want to add it to this object, WDYT?